### DISCUSSION TITLE
Question on the various Q-value

### DISCUSSION THREAD

**Discussion by MagicalMuffins** [vdemichev/DiaNN#357] [Q&A]
I have ran a bunch of DIA runs and primarily interested in the difference of expression of different proteins across my samples. While analyzing the data, I realized that I wanted to filter out the data at the peptide level. From there, I proceeded to get protein expression levels using diann_maxlfq function from diann-rpackage. At this step, I became a little unsure of which Q.Values I should be using. I want to summarize the expression on the gene level and thought using GG.Q.Value would be good; however, I saw on the README for DiaNN say:

> When using MBR and relying on the main report instead of quantitative matrices, use the following q-value filters:
> 
> Lib.Q.Value instead of Global.Q.Value
> When applying a filter to Q.Value that is more stringent than 1% (e.g. Q.Value < 0.001 filter), always apply the same filter to Lib.Q.Value
> Lib.PG.Q.Value instead of Global.PG.Q.Value

I used MBR for my samples and wondered if using Lib.PG.Q.Value would be better. Would it be possible to get some clarifications on the different Q-values in the output and a recommendation on what I should be using?

Thank you very much!

**Comment by vdemichev** [comment]
q-value with 'Global.' or 'Lib.' arer global, all other q-values are run-specific. Please see https://www.nature.com/articles/nmeth.4398 for infor on these.

For summarising on the gene level, equally fine to use PG (=protein group) q-values. Just the resulting effect is very similar.

Best,
Vadim
